v2/Makefile

```diff
@@ -9,11 +9,7 @@ test-stability:
 
 .PHONY: test-quick
 test-quick:
-	go test -count=1 ./...
-
-.PHONY: test
-test-no-race:
-	go test -race ./...
+	go test -count=1 -run=. -bench=. -benchtime=1x ./...
```

Comment:
same here

Reply:
I think I found a solution to this problem. See the update Makefile.

Reply:
do you still want to cleanup duplicated aliases?

Reply:
Yeah I prefer because it already lead to some inconsistencies, but can revert back if you prefer the old names. I might not know the history and all the use cases. Just let me know!